doc/ref/stdlib.html

```diff
@@ -63,7 +63,7 @@ <h4 id="extVar">
       <p>
         If an external variable with the given name was defined, return its string value. Otherwise, raise an error.
       </p>
-      
+
```

Comment:
Sorry about this noise - I have an vim trigger thing that deletes trailing whitespace ...

Reply:
Please don't edit this file manually. Here are the instructions for updating stdlib docs: https://github.com/google/jsonnet#locally-serving-the-website.

Reply:
Ah, that makes a lot more sense - updated.